src/Illuminate/Http/Request.php

```diff
@@ -243,6 +243,16 @@ public function pjax()
         return $this->headers->get('X-PJAX') == true;
     }
 
+    /**
+     * Determine if the request is the result of an prefetch call.
+     *
+     * @return bool
+     */
+    public function prefetch()
+    {
+        return $this->server->get('HTTP_X_MOZ') === 'prefetch' || $this->headers->get('Purpose') === 'prefetch';
```

Comment:
Can this value be case-insensitive?

Reply:
It seems to me that the header names are not case-sensitive.

Reply:
I mean the _value_, not the key.

Reply:
Oh, excuse me, I did not understand.  
Looking for I have for now never a value that was not in lowercase for this variable.

Reply:
We could use https://github.com/laravel/framework/pull/27001 to compare the strings, if it is accepted.